common/test/acceptance/tests/lms/test_account_settings.py

```diff
@@ -266,28 +273,21 @@ def test_email_field(self):
             assert_after_reload=False
         )
 
-        self.verify_events_of_type(
-            username,
-            self.CHANGE_INITIATED_EVENT_NAME,
+        actual_events = self.wait_for_events(
+            event_filter=self.settings_change_initiated_event_filter, number_of_matches=2)
+        self.assert_event_sequences_match(
             [
-                {
-                    u"user_id": long(user_id),
-                    u"setting": u"email",
-                    u"old": email,
-                    u"new": u'me@here.com'
-                },
-                {
-                    u"user_id": long(user_id),
-                    u"setting": u"email",
-                    u"old": email,  # NOTE the first email change was never confirmed, so old has not changed.
-                    u"new": u'you@there.com'
-                }
+                self.settings_change_initiated_event_expectation(
```

Comment:
Nit: I'd probably call this ``expected_settings_change_initiated_event``, as ``expectation`` is a mouthful.